src/main/java/com/hubspot/jinjava/tree/TagNode.java

```diff
@@ -86,15 +87,16 @@ public Tag getTag() {
     return tag;
   }
 
-
   private String reconstructImage() {
     StringBuilder builder = new StringBuilder().append(master.getImage());
 
     for (Node n : getChildren()) {
       builder.append(n.getMaster().getImage());
     }
 
-    builder.append("{% ").append(getEndName()). append(" %}");
+    builder.append(TokenScannerSymbols.TOKEN_EXPR_START).append(TokenScannerSymbols.TOKEN_TAG).append(" ")
```

Comment:
Can we make this just one append with a `String.format` for readability? The StringBuilder makes sense for the above code however makes this bit hard to read.

Reply:
Sure, done. I think it's a bit better now.